admin/ogc-submission

```diff
@@ -0,0 +1,156 @@
+
+The openEO API is also considered as one building block for the emerging OGC Geodatacube API
+
+This specification enhances OGC operations and community involvement by providing a user-centric and less technical approach to big EO cloud processing, backed by an extensive set of pre-defined processes and a large ecosystem of open source servers and clients.
+
+== Evidence of implementation
+
+The following is a list of publically available implementations or deployments of the openEO API specification.
+
+Server deployments (9+):
+* openEO Platform - https://openeo.cloud
+* Copernicus Data Space Ecosystem - https://openeo.dataspace.copernicus.eu
+* EODC - https://openeo.eodc.eu
+* EURAC Research - https://openeo.eurac.edu
+* Google Earth Engine (maintained by a third-party) - https://earthengine.openeo.org
+* Mundialis - https://openeo.mundialis.de
+* Rasdaman - TODO
+* Sentinel Hub - https://openeo.sentinel-hub.com/production
+* VITO - https://openeo.vito.be
+
+Client implementations (5+):
+* JavaScript - https://github.com/open-EO/openeo-js-client
+* Julia - https://github.com/open-EO/openeo-julia-client
+* Python - https://github.com/open-EO/openeo-python-client
+* R - https://github.com/open-EO/openeo-r-client
+* Browser-based User Interface - https://github.com/open-EO/openeo-web-editor
+
+More implementations can be found in the openEO GitHub organization: https://github.com/open-EO
+
+*Date of most recent version:*
+
+Version 1.2.0 of the openEO API was released on May 25, 2023.
+
+*Implementation description:*
+
+The openEO API specification is an open source project released under the Apache 2.0 license and is publicly available via GitHub.
+Most of the server implementations and all of the client implementations mentioned above are also released as open source software. 
+The project is governed by a Project Steering Committee: https://openeo.org/psc/
+
+The API is fully specified using the OpenAPI specification. 
+The openEO API is accompabied by a separately versioned set of process descriptions that can be implemented by server implementations. The processes follow the openEO process description that is defined in the openEO API specification.
+
+*Implementation URL:* 
+https://api.openeo.org
+
+*Is implementation complete?*
+
+* [x] Yes
+* [ ] No
+
+*If not,* what portions of the proposed Community standard are implemented?
+
+////
+Repeat for each implementation. Place a "x" in the appropriate box regarding complete implementation.
+Optionally, provide a narrative description of the extent of implementation of the proposed Community standard for those proposed standards that are very widely used.
+////
+
+== Public availability
+
+Is the proposed Community standard currently publicly available?
+
+* [x] Yes
+* [ ] No
+
+URL:
+https://openeo.org
+
+== Supporting OGC Members
+
+* WWU Münster
+* EURAC
+* VITO
+
+== Intellectual Property Rights
+
+Will the contributor retain intellectual property rights?
```

Comment:
Should we set this to "Yes"?

Reply:
@m-mohr yes I have talked to them, and I don't think this will be an issue. I mentioned this yesterday also in the ESA Tiger Team and also there other entities like TerraDue said that they would support this.

Reply:
@aljacob Is this comment at the right place? I assume you meant to answer regarding the additional supporters, but this thread is about the intellectual property.

Reply:
@m-mohr yes you are right, this was in response regarding the supporting members. Regarding IPR, not really sure how to deal with that.

Reply:
=> Check with OGC staff